packages/redux-dynamic-modules/src/Managers/ModuleManager.ts

```diff
@@ -1,23 +1,15 @@
-import { AnyAction, Dispatch, Middleware, Reducer, ReducersMapObject } from "redux";
-import { SagaMiddleware } from "redux-saga";
-import { IItemManager, IModule, ISagaRegistration } from "../Contracts";
-import { equals as sagaEquals } from "../Utils/SagaComparer";
-import { getReducerManager, getRefCountedReducerManager, IReducerManager } from "./ReducerManager";
-import { getRefCountedManager } from "./RefCountedManager";
-import { getSagaManager } from "./SagaManager";
+import { IModule, IItemManager, IPlugin } from "../Contracts";
+import { AnyAction, ReducersMapObject, Dispatch, Middleware, Reducer } from "redux";
+import { getReducerManager, IReducerManager, getRefCountedReducerManager } from "./ReducerManager";
 
 export interface IModuleManager<State> extends IItemManager<IModule<State>> {
     setDispatch: (dispatch: Dispatch<AnyAction>) => void;
     getReducer: (state: State, action: AnyAction) => State;
 }
 
-export function getModuleManager<SagaContext, State>(
-    sagaMiddleware: SagaMiddleware<SagaContext>,
-    middlewareManager: IItemManager<Middleware>): IModuleManager<State> {
-
+export function getModuleManager<State>(middlewareManager: IItemManager<Middleware>, plugins: IPlugin[]): IModuleManager<State> {

```

Comment:
we should use default params here instead so callers need not pass empty array.

Reply:
This is not exposed